src/abbfreeathome/devices/base.py

```diff
@@ -101,6 +106,17 @@ def get_output_by_pairing(self, pairing: Pairing) -> tuple[str, Any]:
             self.device_id, self.channel_id, pairing.value
         )
 
+    def get_parameter_by_id(self, parameter: Parameter) -> tuple[str, Any]:
```

Comment:
nitpick, maybe name this `get_device_parameter` so you know without reading the code what it's goal is. You're not passing in an `id` and it's returning both the id and the value as a tuple.